build/vsts-ci.yml

```diff
@@ -119,7 +119,7 @@ jobs:
     DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
     DOTNET_MULTILEVEL_LOOKUP: 0
   pool:
-    vmImage: macOS-10.15
+    vmImage: macOS-12
   steps:
   # Work around MacOS Homebrew image/environment bug: https://github.com/actions/virtual-environments/issues/2322#issuecomment-749211076
```

Comment:
I wonder if we still need these workarounds?

Reply:
Looks like it's resolved. I just remove the workaround and kick off a new build